rest_lights.cpp

```diff
@@ -377,20 +377,33 @@ bool DeRestPluginPrivate::lightToMap(const ApiRequest &req, const LightNode *lig
         config["groups"] = groups;
     }
 
-    if (icc)
+    if (icc) // colormode
     {
         const int cc = icc->toNumber();
         QStringList colorModes;
 
         if (cc & 0x10) colorModes.push_back(QLatin1String("ct"));
-        if (cc & 0x04)
+        if (cc & 0x04) // colorloop

```

Comment:
Just a note: Doesn't have to be changed in this PR.

Since the function is one of our very long monsters, it maybe helpful to rename a few variables to be more easy to read through.

E.g.: `itemColorMode` instead of `icc`.

And adding constants for the magic numbers as define or enum for 0x10, 0x04, etc.

Reply:
Done